### PR TITLE
prep-html-skeleton Jacob Yi

### PR DESCRIPTION
**Pull request by Yi-Jacob**
Screen Shot 2022-01-08 at 5 47 34 PM

### CODE REVIEW

File: prep-html-skeleton/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>
+    HTML Skeleton Exercise
+  </title>
```
Comment (thebearingedge) [inline prep-html-skeleton/index.html:6]: Be sure to close your `<head>` element before opening your `<body>` element.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-09 at 2 32 13 PM

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, good. I didn't notice before but you have an extra doctype declaration in your file.

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-09 at 3 10 36 PM

**Pull request merged**